pkgs/development/libraries/gtk/3.x.nix

```diff
@@ -94,6 +101,8 @@ stdenv.mkDerivation rec {
     pkg-config
     python3
     sassc
+    wayland-scanner
```

Comment:
I think this is problematic because wayland doesn't work on darwin. Should wayland-scanner be used dependent on waylandSupport?

Reply:
Yes, it should.

Reply:
Fixed.